style.css

```diff
@@ -3,46 +3,461 @@
 *::after {
   box-sizing: border-box;
   margin: 0;
-  padding: 0;
+  padding: 0 2px;
 }
 
 body {
   display: flex;
   flex-direction: column;
+  font-family: 'Inter', sans-serif;
 }
 
-/* color classes */
-.nav-bar-color {
-  color: #3a4a42;
-}
+/* Mobile Styles */
+@media screen and (max-width: 767px) {
+  .container {
```

Comment:
[OPTIONAL] As you know you using a mobile-first approach and in a mobile-first approach we are not using any media query you just write your CSS without a media query and we just use a media query for the desktop version so kindly fix this I think the previous reviewer missed this requirement that's why I give you an `optional` but this is `recommended` so kindly `don't` skip this `requirement` :slightly_smiling_face: